docs/InstallGuide.md

```diff
@@ -58,15 +46,14 @@ Modify the following section in /etc/openvnet/vnmgr.conf on the vnmgr node.
       addr {
         protocol "tcp"
         host "192.168.100.2"
```

Comment:
This IP address is not mentioned before. Why is it 192.168.100.2? Since we are installing everything in one machine, we should just use 127.0.0.1. That way it will work regardless of what IP address the reader is using.